dev-packages/e2e-tests/test-applications/nestjs-basic/tests/errors.test.ts

```diff
@@ -65,7 +65,7 @@ test('Does not send HttpExceptions to Sentry', async ({ baseURL }) => {
   await transactionEventPromise400;
   await transactionEventPromise500;
 
-  await new Promise(resolve => setTimeout(resolve, 10000));
+  await fetch(`${baseURL}/flush`);
```

Comment:
```suggestion
  await (await fetch(`${baseURL}/flush`)).text();
```

Let's await the full response body to ensure the endpoint completed. Same for the others.